tip-wlan/charts/kafka/templates/secret.yaml

```diff
@@ -1,8 +1,18 @@
+---
 apiVersion: v1
 kind: Secret
 metadata:
   name: {{ include "common.fullname" . }}-certs
   namespace: {{ include "common.namespace" . }}
+  labels:
+    app: {{ template "common.name" . }}
+    chart: {{ template "common.chart" . }}
+    release: {{ .Release.Name | quote }}
 type: Opaque
 data:
-{{ tpl (.Files.Glob "resources/config/certs/*").AsSecrets . | indent 2 }}
\ No newline at end of file
+  truststore_creds: {{ .Values.creds.sslTruststorePassword | b64enc }}
```

Comment:
This is good. Thank you for removing the files and instead getting the values from secrets